mongo/test/org/immutables/mongo/bson4gson/GsonCodecsTest.java

```diff
@@ -22,7 +25,7 @@ public void reflectiveTypeAdapter() {
   }
 
   @Test
-  public void dateCodec() throws IOException {
+  public void typeAdapterFromCodec() throws IOException {
```

Comment:
I think keeping the old name (`dateCodec`) makes more sense

Reply:
I went with `typeAdapterFromDateCodec`